src/decoderbufs.c

```diff
@@ -347,15 +346,6 @@ static void set_datum_value(Decoderbufs__DatumMessage *datum_msg, Oid typid,
       datum_msg->datum_double = (double) (timetz->time + (timetz->zone * 1000000.0));
       datum_msg->datum_case = DECODERBUFS__DATUM_MESSAGE__DATUM_DATUM_DOUBLE;
       break;
-    case INTERVALOID:
```

Comment:
@jpechane, how does that impact updating? In particular, will both combinations work as expected:

* updated decoderbufs with old connector JAR
* old decoderbufs with updated connector JAR

?

Reply:
Old decoderbuf with new connector -> string with seconds only, numeric unchanged
New decoderbuf with old connector - a null value is sent

Reply:
Ok, added an [entry](https://issues.jboss.org/browse/DBZ-1498?focusedCommentId=13796671&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13796671) for the migration notes.